lib/instrumental.js

```diff
@@ -238,19 +238,25 @@ exports.init = function instrumental_init(startup_time, config, events) {
     }
 
     // do no filtering by default
-    if(typeof(config.instrumental.metricFilters) === 'undefined'){
-      metricFilters = [];
+    if(typeof(config.instrumental.metricFiltersExclude) === 'undefined'){
+      metricFiltersExclude = [];
     } else {
-      metricFilters = config.instrumental.metricFilters;
-
-      keepMetric = function(metric){
-        metricName = metric.split(" ")[1];
-        return !metricFilters.some(function(filter) {
-          return metricName.match(filter);
-        });
-      };
+      metricFiltersExclude = config.instrumental.metricFiltersExclude;
     }
 
+    // do no filtering by default
+    if(typeof(config.instrumental.metricFiltersInclude) === 'undefined'){
+      metricFiltersInclude = [/.*/];
+    } else {
+      metricFiltersInclude = config.instrumental.metricFiltersInclude;
+    }
+
+    keepMetric = function(metric){
+      metricName = metric.split(" ")[1];
+      return (!metricFiltersExclude.some(function(filter) { return metricName.match(filter);}) &&
+              metricFiltersInclude.some(function(filter) { return metricName.match(filter);}))
```

Comment:
Not a blocker, but this might be slightly more correct using the regex [`test`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) method instead of [`match`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match). `test` returns a boolean, while `match` returns `null` or an array of matches. Those are obviously evaluated as falsey and truthy respectively, so the current implementation works just fine.

e.g:

```
return filter.test(metricName);
```